editor/plugins/shader_editor_plugin.h

```diff
@@ -172,7 +208,7 @@ class ShaderEditorPlugin : public EditorPlugin {
 	virtual void make_visible(bool p_visible) override;
 	virtual void selected_notify() override;
 
-	ShaderEditor *get_shader_editor() const { return shader_editor; }
+	ShaderEditor *get_shader_editor(const Ref<Shader> &p_for_shader);
```

Comment:
Why is this method no longer const?

Reply:
It does not make sense to be const, since you are requesting internal state of the function.